### PR TITLE
chore: extract nano id zod schema to a reusable util

### PR DESCRIPTION
**Pull request by sean-brydon**
I havent tested this worked and just used github.dev - but here is what i mean 

(From discord DMs)

### CODE REVIEW

File: packages/utils/index.ts
Comment (sean-brydon) [inline packages/utils/index.ts:1]: We might not want this to live in this utils file - might be best adding a `zod-utils` or `validation` file somewhere in utils or lib. 

See Cal.com repo - [here](https://github.com/calcom/cal.com/blob/b76a2a4019cdfab5fe2da0d5e7347893dd38a726/packages/prisma/zod-utils.ts#L4)

Comment (McPizza0) [reply packages/utils/index.ts:1]: I think its ok for now
if we increase number of zod schemas we can create a separate package

**Review by McPizza0 — APPROVED** [review]
lgtm

File: packages/utils/index.ts
Comment (skushagra9) [reply packages/utils/index.ts:1]: yes

**Pull request merged**